test/unit-test/commander/test_commander_set_mnemonic_passphrase_enabled.c

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <cmocka.h>
+
+#include <test_commander.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmissing-prototypes"
+#pragma GCC diagnostic ignored "-Wimplicit-function-declaration"
+
+bool __wrap_workflow_confirm(const char* title, const char* body, bool accept_only)
+{
+    check_expected(title);
+    check_expected(body);
+    check_expected(accept_only);
+    return mock();
```

Comment:
You could use `mock_type(bool)` here I believe.

Reply:
TIL